apps/platform-app/src/app/[locale]/(profile)/(routes)/my-profile/(components)/LanguageInput/LanguageInput.tsx

```diff
@@ -1,10 +1,10 @@
 import styles from '@/components/TechStackInput/TechStackInput.module.scss'
 import languages from '@/data/languages'
```

Comment:
**nitpick**: Consider ordering imports by external libraries first, followed by internal modules for consistency.